scripts/pages/settings/speedometer.js

```diff
@@ -716,7 +695,7 @@ class RangeColorProfiles {
 
 	static makeColorProfileNamePopup(prefilledText, OKBtnText, callback) {
 		const profileNames = [];
-		for (const profileName of Object.keys(RangeColorProfiles.objectList)) profileNames.push(profileName);
+		for (const [profileName, _] of RangeColorProfiles.profileObjectMap) profileNames.push(profileName);
```

Comment:
Another `.map` case! Also can't you just leave out the underscore here? i.e. `for (const [profileName] of RangeColorProfiles.profileObjectMap)`

Reply:
Realized that this is literally just `profileObjectMap.keys()` lmao